Source/SwiftLintFramework/Models/Configuration.swift

```diff
@@ -213,16 +213,11 @@ public struct Configuration {
             self.fileGraph = fileGraph
             setCached(forIdentifier: cacheIdentifier)
         } catch {
-            let errorString: String
-            switch error {
-            case let ConfigurationError.generic(message):
-                errorString = "SwiftLint Configuration Error: \(message)"
-
-            case let YamlParserError.yamlParsing(message):
-                errorString = "YML Parsing Error: \(message)"
-
-            default:
-                errorString = "Unknown Error"
+            guard let errorString = errorMessage(for: error, hasCustomConfigurationFiles: hasCustomConfigurationFiles)
+            else {
+                // Fall back to default
+                self.init(rulesMode: rulesMode, cachePath: cachePath)
```

Comment:
I'd prefer if SwiftLint would at least print about what it is doing here (not as a warning, but as a regular print). That would require bringing back the `silent` flag as a parameter of this initializer.

However, this print is not so important and maybe it's better to not increase complexity by adding this (`silent`-conditional) print. Do what you think is the better solution ;)

Reply:
I'd rather SwiftLint not print at all in the common case that you want to lint a file or files without a configuration.